docs/guide-configs.md

```diff
@@ -21,39 +21,312 @@ Below you will find pre-built configuration files for the following TRaSH Guides
-[web-1080p-v4-config]: https://raw.githubusercontent.com/recyclarr/config-templates/master/sonarr/web-1080p-v4.yml
-[web-2160p-v4-config]: https://raw.githubusercontent.com/recyclarr/config-templates/master/sonarr/web-2160p-v4.yml
-[anime-sonarr-v4-config]: https://raw.githubusercontent.com/recyclarr/config-templates/master/sonarr/anime-sonarr-v4.yml
-[french-web-1080p-multi-v4-config]: https://raw.githubusercontent.com/recyclarr/config-templates/master/sonarr/french-web-1080p-multi-v4.yml
-[french-web-1080p-vostfr-v4-config]: https://raw.githubusercontent.com/recyclarr/config-templates/master/sonarr/french-web-1080p-vostfr-v4.yml
-[french-anime-multi-v4-config]: https://raw.githubusercontent.com/recyclarr/config-templates/master/sonarr/french-anime-multi-v4.yml
-[french-anime-vostfr-v4-config]: https://raw.githubusercontent.com/recyclarr/config-templates/master/sonarr/french-anime-vostfr-v4.yml
+| Radarr                                                             | Sonarr v4                                              |
+| ------------------------------------------------------------------ | ------------------------------------------------------ |
+| [HD Bluray + WEB](#hd-bluray--web)                                 | [WEB-1080p](#web-1080p-v4)                             |
+| [UHD Bluray + WEB](#uhd-bluray--web)                               | [WEB-2160p](#web-2160p-v4)                             |
+| [Remux + WEB 1080p](#remux--web-1080p)                             | [Anime (Sonarr)](#anime-sonarr-v4)                     |
+| [Remux + WEB 2160p](#remux--web-2160p)                             | [French WEB-1080p MULTi](#french-web-1080p-multi-v4)   |
+| [Anime (Radarr)](#anime-radarr)                                    | [French WEB-1080p VOSTFR](#french-web-1080p-vostfr-v4) |
+| [French HD Bluray + WEB MULTi](#french-hd-bluray--web-multi)       | [French Anime MULTi](#french-anime-multi-v4)           |
+| [French HD Bluray + WEB VOSTFR](#french-hd-bluray--web-vostfr)     | [French Anime VOSTFR](#french-anime-vostfr-v4)         |
+| [French UHD Bluray + WEB MULTi](#french-uhd-bluray--web-multi)     |                                                        |
+| [French UHD Bluray + WEB VOSTFR](#french-uhd-bluray--web-vostfr)   |                                                        |
+| [French Remux + WEB 1080p MULTi](#french-remux--web-1080p-multi)   |                                                        |
+| [French Remux + WEB 1080p VOSTFR](#french-remux--web-1080p-vostfr) |                                                        |
+| [French Remux + WEB 2160p MULTi](#french-remux--web-2160p-multi)   |                                                        |
+| [French Remux + WEB 2160p VOSTFR](#french-remux--web-2160p-vostfr) |                                                        |
+
+---
+
+## Radarr
+
+### HD Bluray + WEB
```

Comment:
There's one thing I think you should do here. I feel that it is important because the auto-generated anchors for these headers are fickle. It would be nice to rename headings without having to worry about broken links.

Docusaurus supports explicit anchors. It's a bit tedious to set up since we have so many, but after that you don't need to touch it.

Example:

```md
### HD Bluray + WEB {#hd-bluray-web}
```

Then in your table of links:

```md
[HD Bluray + WEB](#hd-bluray-web)
```

I think this should be done explicitly for every single heading that shows up in the table at the top. I know this is a tedious ask; I'm sorry!

Reply:
> There's one thing I think you should do here. I feel that it is important because the auto-generated anchors for these headers are fickle. It would be nice to rename headings without having to worry about broken links.
> 
> Docusaurus supports explicit anchors. It's a bit tedious to set up since we have so many, but after that you don't need to touch it.
> 
> Example:
> 
> ```
> ### HD Bluray + WEB {#hd-bluray-web}
> ```
> 
> Then in your table of links:
> 
> ```
> [HD Bluray + WEB](#hd-bluray-web)
> ```
> 
> I think this should be done explicitly for every single heading that shows up in the table at the top. I know this is a tedious ask; I'm sorry!

I'm actually glad you pointed this out, as I didn't realise you could do this. Agreed, much better, and have committed this change across linked headers.